doc/user.rst

```diff
@@ -62,27 +62,26 @@ In any case, you need:
 Each plugin usually needs a codec library, which you also need to
 install. Check the :doc:`plugins` for details about required libraries
 
-For example, the following installs a fairly complete list of build dependencies on Debian Jessie:
+For example, the following installs a fairly complete list of build dependencies on Debian Buster:
 
 .. code-block:: none
 
-    apt install g++ \
+    apt install meson g++ \
       libpcre3-dev \
       libmad0-dev libmpg123-dev libid3tag0-dev \
-      libflac-dev libvorbis-dev libopus-dev \
+      libflac-dev libvorbis-dev libopus-dev libogg-dev \
       libadplug-dev libaudiofile-dev libsndfile1-dev libfaad-dev \
-      libfluidsynth-dev libgme-dev libmikmod2-dev libmodplug-dev \
+      libfluidsynth-dev libgme-dev libmikmod-dev libmodplug-dev \
       libmpcdec-dev libwavpack-dev libwildmidi-dev \
-      libsidplay2-dev libsidutils-dev libresid-builder-dev \
+      libsidplayfp-dev libsidutils-dev libresid-builder-dev \
       libavcodec-dev libavformat-dev \
-      libmp3lame-dev libtwolame-dev libshine-dev \
```

Comment:
Why remove twolame and shine?

Reply:
Not certain about `twolame`, it provides support for "MPEG Audio Layer 2" but I think libmad also provides it.

Regarding `shine` we only enable it on armhf in the debian build but your right, it should stay in the user doc.

Reply:
I'm ok with removing useless libraries, but not hidden inside a commit pretending to do the Buster update.